### PR TITLE
Custom MOTD for EmbassyOS

### PR DESCRIPTION
**Pull request by k0gen**
As mentioned in #1083 This is the initial work to get things going ...

### CODE REVIEW

**Review by kn0wmad — COMMENTED** [review]
LGTM

File: build/00-embassy [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+printf "\n"
+printf "Welcome to\n"
+cat << "ASCII"
+,---.     |                        ,---.,---.
+|--- ,-.-.|---.,---.,---.,---.,   .|   |`---.
+|    | | ||   |,---|`---.`---.|   ||   |    |
+`---'` ' '`---'`---^`---'`---'`---|`---'`---'
+                              `---'
+ASCII
+printf "   %s (%s %s)\n" "$(uname -o)" "$(uname -r)" "$(uname -m)"
```
Comment (ProofOfKeags) [inline build/00-embassy:11]: add to this `EmbassyOS v$(embassy-cli version)) - $(embassy-cli git-info)`

Comment (k0gen) [reply build/00-embassy:11]: You mean something like this?
```
ASCII
printf "   %s (%s %s)\n" "$(uname -o)" "$(uname -r)" "$(uname -m)"
printf "   $(embassy-cli --version) - $(embassy-cli git-info)\n"
```
will result into:

Screenshot from 2022-01-24 22-40-16

Comment (ProofOfKeags) [reply build/00-embassy:11]: Yes.

Comment (ProofOfKeags) [reply build/00-embassy:11]: There is currently a bug in embassy-cli that causes version not to work but that's exactly what I mean.

**Review by ProofOfKeags — APPROVED** [review]

**Review by kn0wmad — APPROVED** [review]

**Pull request merged**